include/z64.h

```diff
@@ -184,10 +184,22 @@ typedef struct {
     /* 0x0104 */ Vec3f  curDistortionOrientation;
     /* 0x0110 */ Vec3f  curDistortionScale;
     /* 0x011C */ u16    normal; // used to normalize the projection matrix
-    /* 0x0120 */ s32    flags;
+    /* 0x0120 */ s32    dirtyFlags;
```

Comment:
Where does `dirty` come from? Is that a term used in computer graphics/game engine? It's not something I've heard of before

Reply:
Yeah, I'd say it's fairly commonly used. The pattern is usually that you want to "update" something from several places, but it would be a waste to actually do all those updates, so you just set the flag and then whatever code actually reads the value can look at the flag and do the actual update if necessary before using it.

It basically changes N expensive updates into N cheap flag sets and 1 expensive update.

https://stackoverflow.com/questions/13503243/what-does-dirty-flag-dirty-values-mean

Reply:
Changed back to `flags` bc841becfc21a7818d20832e6e50ff07589f081a